drivers/mipi_dsi/dsi_mcux.c

```diff
@@ -39,6 +39,9 @@ LOG_MODULE_REGISTER(dsi_mcux, CONFIG_MIPI_DSI_LOG_LEVEL);
 #define DSI_DPHY_PLL_CO_MIN 0
 #define DSI_DPHY_PLL_CO_MAX 3
 
+/* MAX DSI TX payload */
+#define DSI_TX_MAX_PAYLOAD_BYTE (64U * 4U)
```

Comment:
where does this come from? it seems like according the 595 RM, you can send 2^15 pixels in one dsi packet, but maybe I am reading it upside down or something

Reply:
This limit is being enforced by the SDK driver- but I believe it might come from the size of the TX FIFO for APB packets. The TX FIFO RAM is "64x32", but the RM does not clarify if the size is in bytes or bits. If the TX FIFO RAM size is given in bits, than the limit enforced by the SDK here makes sense: 
![image](https://github.com/zephyrproject-rtos/zephyr/assets/34665051/d1615328-0359-474b-bac0-85dd0c689421)